docs/pages/index.mdx

```diff
@@ -2,39 +2,134 @@ import ImageRow from "@/components/ImageRow";
+
+#### Registry Service
+
+Whenever a new Origin or Staging device is created and added to a federation, its first step is to register itself with the *Registry*, which acts as the federation's locus of trust. In the case of Origins, the process of registration entails sending the Registry the namespace prefix the Origin exports, along with the Origin's public key and a variety of other bookkeeping information. After the Registry and the Origin have performed a handshake that proves the Origin owns the corresponding private key, the Registry stores the information in a persistent database.
+
+This process serves two purposes -- first, whenever the Origin re-advertises with the federation's Director, the Director can verify the authenticity of those advertisements through public/private key asymmetric cryptography by looking at the Registry's stored public key for that Origin and namespace. Second, the Registry's persistent database prevents other Origins from registering namespaces under an already-registered namespace without first proving they're allowed to do so by the namespace owner (i.e. the entity that possesses the appropriate private key).
+
+## Making Bytes Accessible and Moving Them -- A First Look Under The Hood
+
+This section provides a simplified example of how data is made accessible and moved within the OSDF. In particular, it elides the OSDF’s Staging/Caching infrastructure and any discussion of authorization tokens.
+
+Pelican serves two sides of the same coin -- Data owners who want to federate their data from wherever it lives natively, and data consumers who want to access and compute on data wherever they need it.
+
+
+<ImageRow alt={"Pelican and OSDF"} src={"/pelican/arch-repo-and-consumer.png"}>
+	The federation's core goal is connecting data owners and data consumers.
+</ImageRow>
+
+
+As such, the primary prerequisite for data to be moved via a Pelican federation is for a data owner to make their data accessible to the federation. This happens when an Origin is placed in front of the repository and registered with the federation. While federations like the OSDF *may* wish to control or filter any Origin registrations to vet the data they make available, this example assumes the Origin's registration is automatically approved. The red arrow in the following graphic represents the vetting/approval step, should the federation require it.
```

Comment:
I didn't find any red arrows in the graphics in this PR. Should this be updated?

Also, I'm a bit hesitated to give out the details for approval in the overview doc, but it's a good topic for the federation overview page for sure.

Reply:
Thanks for catching the image, I'll update it.

As for the approval step, I think it's worth mentioning because the majority of people reading this for the foreseeable future will be registering with the OSDF.